Dockerfile

```diff
@@ -961,13 +961,16 @@ RUN \
 # Generate macros for the target.
 
 FROM sdk as sdk-macros
-ARG ARCH
 
 COPY macros/* /tmp/
 
 WORKDIR /tmp
 RUN \
-  cat ${ARCH} shared rust cargo > /etc/rpm/macros.bottlerocket
+  for arch in x86_64 aarch64 ; do \
```

Comment:
Will this change require rework on how the packages are compiled in Bottlerocket?

Reply:
> Will this change require rework on how the packages are compiled in Bottlerocket?

It's transparent at the repo level but does require the supporting changes in `twoliter`.